.github/workflows/codeql-action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'CodeQL'
```

Comment:
should it be any different than what we have in [1] ?

[1] https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/master/.github/workflows/codeql.yml